src/olympia/devhub/urls.py

```diff
@@ -241,7 +257,7 @@
         # Old LWT Theme submission.
         re_path(
             r'^theme/submit/?$',
-            lambda r: redirect('devhub.submit.agreement'),
+            lambda r: redirect('devhub.submit.theme.agreement'),
             name='devhub.themes.submit',
```

Comment:
Not entirely the fault of this patch, but we now have `devhub.themes.submit` and `devhub.submit.theme.*` urls.  Rename the old LWT one to something obviously deprecated, but under the same naming scheme?

Reply:
722f7d6038b545be69edb900f74f9837bce24bbe